### PR TITLE
changed the author name to devstrons and changed the grammar mistake.

### PR DESCRIPTION
**Pull request by ujjwalvats16**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ A repository that includes Hello World from popular languages added by our commu
...
-3. Add this content to your newly created file. Add your GitHub username in the `author` field. Just remove the `\` in the code backticks.
+3. Add this content to your newly created file. Add your GitHub username in the `author` field. Just remove the `\` in the code backtick.
 
 ```markdown
 ---
 lang: Java
 syntax: java
 title: Java is so cool!
-author: Panquesito7
+author: DEVSTRONS
```
Comment (vsnthdev) [inline README.md:16]: please make this lowercase as usernames and URLs should be lowercase

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ A repository that includes Hello World from popular languages added by our commu
 
 1. Fork the repository, and create a branch named `<language_to_add>`. In this example, we're adding **Java** to the list.
```
Comment (vsnthdev) [inline README.md:7]: Creating a branch isn't required. They're supposed to add the language directly into the `main` branch.

File: README.md [hunk 3]
```diff
@@ -6,14 +6,14 @@ A repository that includes Hello World from popular languages added by our commu
...
 2. Create a file in the `lang` folder with the name of the language you want to add (e.g.: `java.md`).
-3. Add this content to your newly created file. Add your GitHub username in the `author` field. Just remove the `\` in the code backticks.
+3. Add this content to your newly created file. Add your GitHub username in the `author` field. Just remove the `\` in the code backtick.
```
Comment (vsnthdev) [inline README.md:9]: "backticks" is the correct grammar since there are multiple of them.

**Review by vsnthdev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Panquesito7) [reply README.md:7]: That means: they should create a new branch in their repository, and get merged to the `main` branch. And that branch it's not going to be in this repository, it's going to be in their fork. 🙂

File: README.md (hunk 1 above)
Comment (Panquesito7) [inline README.md:16]: I used `Panquesito7` here as an example of myself adding the Java language, so you should probably revert it or add another username.

**Pull request merged**